### PR TITLE
[Dash 4] Application flow

### PR DESCRIPTION
**Pull request by NotMyLyfe**

### CODE REVIEW

File: src/components/ApplicationForm/About/About.module.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    input{
+        border: none;
+        background-color: colors.getColor(neutral-400, null, 0.18);
+        color: colors.getColor(neutral-400);
+        &:disabled{
+            background-color: colors.getColor(neutral-50, null, 0.08);
+            color: colors.getColor(neutral-50);
+        }
+        &:focus{
+            color: colors.getColor(shades-100);
```
Comment (NotMyLyfe) [inline src/components/ApplicationForm/About/About.module.scss:16]: Unsure what should be included here, not specified in the main design specification. Currently, it transitions to a white-background field, and as such a colour that is viewable was included; however, it is up to design what should be included.

File: src/components/ApplicationForm/About/About.module.scss [hunk 2]
```diff
@@ -4,13 +4,16 @@
 @use '../../../styles/vars.module';
 
 .field{
+    label:has(+ input:disabled){
```
Comment (NotMyLyfe) [inline src/components/ApplicationForm/About/About.module.scss:7]: This isn't entirely supported by all browsers, namely Firefox. Nonetheless, this is probably the best and only way to do such as task.

File: src/components/ApplicationForm/About/About.module.scss (hunk 1 above)
Comment (BlazingAsher) [reply src/components/ApplicationForm/About/About.module.scss]: Changed behaviour to match landing page

**Review by BlazingAsher — APPROVED** [review]
LGTM

**Pull request merged**